### DISCUSSION TITLE
SummerCamp 2023 - Lightning talks

### DISCUSSION THREAD

**Discussion by tormath1** [sre-france/meetups#135] [Show and tell]
Propose here your lightning talks :arrow_down: - 5 minutes slot :)

**Comment by guilhem** [comment]
Kube-Native application: one API to rule them all

**Comment by ericacalogero** [comment]
Tech stack of an Out of Home (OOH) advertising company.

**Comment by unixfox** [comment]
IPV6 : Why it's important.

**Comment by tormath1** [comment]
Composable images with [systemd-sysext](https://www.freedesktop.org/software/systemd/man/systemd-sysext.html)

**Comment by ChapChap** [comment]
Some terraform don'ts to avoid headache 🤕
